tests/Functional/tests/escape/escape.rst

```diff
@@ -1 +1,3 @@
 Testing that this is escaped: <script>
+
+\And escape \characters are ig\nored, but this one isn't: \\stdClass
```

Comment:
Escape characters are not rendered according to the Sphinx parser. This was also required to make `\1. Single line paragraphs starting with a number may use an escape.` pass.